diff --git a/node_modules/@types/react-native-calendars/index.d.ts b/node_modules/@types/react-native-calendars/index.d.ts
index 4075af6..78de8bb 100644
--- a/node_modules/@types/react-native-calendars/index.d.ts
+++ b/node_modules/@types/react-native-calendars/index.d.ts
@@ -275,6 +275,11 @@ export interface CalendarBaseProps {
      */
     renderArrow?: (direction: 'left' | 'right') => React.ReactNode;
 
+    /**
+     *  Replace default month title with custom one (recevied 'monthString')
+     */
+    renderMonthTitle?: (monthString: string) => React.ReactNode;
+
     /**
      *  Show week numbers to the left. Default = false
      */
