diff --git a/node_modules/@types/react-native-calendars/index.d.ts b/node_modules/@types/react-native-calendars/index.d.ts
index 4075af6..05a1bcc 100644
--- a/node_modules/@types/react-native-calendars/index.d.ts
+++ b/node_modules/@types/react-native-calendars/index.d.ts
@@ -275,6 +275,16 @@ export interface CalendarBaseProps {
      */
     renderArrow?: (direction: 'left' | 'right') => React.ReactNode;
 
+    /**
+     *   Replace default header with custom one (receives 'onPressLeft', 'onPressRight', 'monthString', and 'indicator')
+     */
+    renderHeader?: (
+        onPressLeft: () => void,
+        onPressRight: () => void,
+        monthString: string,
+        indicator: JSX.Element,
+    ) => React.ReactNode;
+
     /**
      *  Show week numbers to the left. Default = false
      */
